### PR TITLE
[error-pages] Update kubernetes ingress api to `v1`

### LINKED ISSUE #454: [error-pages] Deprecated networking API
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**:
helm: 3.7.2
K8s: 1.22

**Which chart in which version**:
error-pages 0.1.2

**What happened**:
The ingress networking resource is defined with the deprecated API version `networking.k8s.io/v1beta1`.
This API version is deprecated in kubernetes v1.22

**What you expected to happen**:
Change ingress networking resource with appropriate API version `networking.k8s.io/v1` for kubernetes v1.22

**How to reproduce it** (as minimally and precisely as possible):
template Chart with Values.ingress.enabled true

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by theonlysinjin**
#### What this PR does / why we need it:
We are upgrading our kubernetes cluster to v1.22 and the current chart uses a (now) deprecated kubernetes API
You can see here that it is [being deprecated](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122) and then [in this list](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#ingress-v1-networking-k8s-io) you can see that it is not being included at all

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #454 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/error-pages/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Kubernetes error pages for traefik
 name: error-pages
-version: 0.1.2
+version: 0.1.3
```
Comment (rpahli) [inline charts/error-pages/Chart.yaml:5]: pls update the version to 1.0.0 as this is a breaking change

**Review by rpahli — CHANGES_REQUESTED** [review]

File: charts/error-pages/Chart.yaml (hunk 1 above)
Comment (theonlysinjin) [reply charts/error-pages/Chart.yaml]: Hey @rpahli , thanks, have done so.

**Review by rpahli — DISMISSED** [review]

**Comment by rpahli** [comment]
Some of your commit are not signed. Can you pls sign your commits.

**Comment by theonlysinjin** [comment]
Thanks @rpahli , sorry for the delay here.
I signed that one commit and seems ok now

**Comment by theonlysinjin** [comment]
Thanks @rpahli sorry for the delay,
Signed that one commit and rebased.

**Review by rpahli — APPROVED** [review]

**Pull request merged**